pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -930,6 +930,9 @@ let
 
       SCHED_CORE = whenAtLeast "5.14" yes;
 
+      CONFIG_LRU_GEN = { optional = true; tristate = whenAtLeast "6.1" "y"; };
+      CONFIG_LRU_GEN_ENABLED =  { optional = true; tristate = whenAtLeast "6.1" "y"; };
+
```

Comment:
```suggestion
      LRU_GEN = whenAtLeast "6.1" yes;
      LRU_GEN_ENABLED =  whenAtLeast "6.1" yes;

```

Reply:
Options don't use `CONFIG_`  prefix.
I'm not sure why other code has it same way as you have written, if you need it to be optional then it should rather be `whenAtLeast "6.1" (option yes)` or `option yes`, do that if it breaks something.

Reply:
that worked, thanks!